bucket/flat-assembler.json

```
@@ -1,10 +1,13 @@
 {
-    "version": "1.73.29",
+    "version": "1.73.30",
     "description": "Open source assembly language compiler",
     "homepage": "https://flatassembler.net/",
-    "license": "Unknown",
-    "url": "https://flatassembler.net/fasmw17329.zip",
-    "hash": "6995d164069dd45d45a8228440935ef38e444a35d700af7a339c67b3d5c67d23",
+    "license": {
+        "identifier": "Unknown",
```

```suggestion
        "identifier": "Freeware",
```
I think the license is marked as **unknown** because "the author of the manifest" could not find the license document. It is actually **freeware**.

```
This program is free for commercial and non-commercial use as long as
the following conditions are adhered to.
```